src/LibraryManager/Providers/Unpkg/NpmPackageSearch.cs

```diff
@@ -179,9 +181,11 @@ private static async Task<IEnumerable<string>> GetPackageNamesFromSimpleQueryAsy
                             if (packageEntry != null)
                             {
                                 string currentPackageName = packageEntry["name"].ToString();
-                                if (!String.IsNullOrWhiteSpace(currentPackageName))
+                                string currentPackageVersion = packageEntry["version"].ToString();
```

Comment:
This seems really close to what we do in CreatePackageInfoAsync.  Maybe we can consolidate with that and return NpmPackageInfo instead of this tuple hack?